### DISCUSSION TITLE
Play videos inside app

### DISCUSSION THREAD

**Discussion by datrix88** [Radiokot/photoprism-android-client#26] [Ideas]
Hi! This app Is really awesome!

It would be great if there was the possibility to play videos directly inside the app without having to download them, as already happens with the web interface.

Thanks in Advance and congratulations for the wonderful app :)

**Comment by Radiokot** [comment]
Hi, thanks for checking out the app!
I'm working on the video content preview (videos, live photos, animations). Will be added soon 😉

**Comment by Radiokot** [comment]
Added in [1.5.0](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.5.0(8))

**Comment by ippocratis** [comment]
Is there any sort of limitation on video/live photo/animation preview from within the app?
I cant seem to play or preview them
I can see a play button centered in a black screen but hitting play only spins the button for some seconds and duration stays at 00:00

**Reply by Radiokot** [reply]
Oh shit, I'm sure this is because the video streamer does not support mTLS... Have to add this 🙂

**Reply by Radiokot** [reply]
Thank you for reporting, Ippokratis!

**Comment by ippocratis** [comment]
In app Video with servers behind mtls works fine in 1.5.1
Thanks so much

**Reply by Radiokot** [reply]
That's awesome 👌
